arkivering-end-to-end-tests/src/test/kotlin/no/nav/soknad/arkivering/arkiveringendtoendtests/EndToEndTests.kt

```diff
@@ -309,6 +317,68 @@ class EndToEndTests {
 		pollAndVerifyDataInFileStorage(uuid, 1)
 	}
 
+	@Test
+	fun `Put input event on Kafka when Soknadsarkiverer is down - will start up and send to Joark`() {
+		val key = UUID.randomUUID().toString()
+		val uuid = UUID.randomUUID().toString()
+		val dto = createDto(uuid, uuid)
+		setNormalJoarkBehaviour(dto.innsendingsId)
```

Comment:
setNormalArkivBehavior
bør bli kvitt joark benevnelsen. Det er appen "dokarkiv" vi faktisk forholder oss til -> kort for dokumentarkiv